lib/filters.js

```diff
@@ -0,0 +1,66 @@
+var _ = require('lodash'),
+    moment = require('moment');
+
+/**
+ * filter items
+ * Add your filters to the fi obj as methods. See examples below.
+ * "fs" being short allowing tidier composition
+ */
+var fi = {};
+
+/**
+ * creates rearranges values and creates new date object
+ * @param  {String} d   A date string (must be) formatted (d)d/(m)m/yyy - in parens means optional
+ * @return {String}     a javascript date string
+ */
+fi.newDate = function date(d) {
+	var dateArr = d.split('/');
+	return dateArr.length === 3 ? new Date(dateArr[2], parseInt(dateArr[1]) - 1, dateArr[0]) : NaN;
+};
+
+/**
+ * returns a standard gov.uk date from a string using momentjs
+ * moment documentation: http://momentjs.com/docs/
+ * @method function
+ * @param  {string} d date e.g 09/12/1981 or 9-12-1981
+ * @param  {string} f moment.js format string (to override the default if needed)
+ * @return {string} date string as per the current gov.uk standard 09/12/1981 -> 09 December 1981
+ */
+fi.formatDate = function(d,f) {
+  return moment(fi.newDate(d)).locale('en-gb').format(f ? f : 'LL');
+};
+
+/**
+ * logs an object in the template to the console on the client.
+ * @param  {Any} a any type
+ * @return {String}   a script tag with a console.log call.
+ * @example {{ "hello world" | log }}
+ * @example {{ "hello world" | log | safe }}  [for environments with autoescaping turned on]
```

Comment:
Looks like we have two options. 

## 1. disable auto-escaping by default.
This will help us when it comes to filters because it *appears* there is no way to get around auto-escaping, it will always get escaped on the way out.

If we did this then anybody wishing to escape content could either use the Nunjucks filters inline or on a block of content. 

**Inline on a string**
``` {{ '<h1>Escape this</h1>' | escape }}``` => ```&lt;h1&gt;Escape this&lt;/h1&gt;```

**On a chunk of content/mark-up**
```
{% set sampleHTMLChunk %}
<div>
<h1>This HTML will be escaped</h1>
</div>
{% endset %}

{{ sampleHTMLChunk | escape }}
```
Which would produce
```
&lt;div&gt;
&lt;h1&gt;This HTML will be escaped&lt;/h1&gt;
&lt;/div&gt;
```
## 2. Keep auto escaping

We keep auto-escaping as-is and use ```safe``` in the same way we'd have used ```escape``` above.